test/service/test_response.py

```diff
@@ -1088,31 +1094,6 @@ def test_filter_by_contentDescription(self):
                   for file in hit['files']]
         self.assertEqual(actual, expected)
 
-    def test_translated_facets(self):
-        """
-        Test that response facets values are correctly translated back to the
-        correct data types and that the translated None value is not present.
-        """
-        url = self.base_url.set(path='/index/samples',
```

Comment:
This hits the samples endpoint. Your facet test hits the projects endpoint. Given that, I'm not sure I'd call this test redundant.